src/main/java/com/logistic/project/model/Order.java

```diff
@@ -1,4 +1,10 @@
 package com.logistic.project.model;
 
+import lombok.Data;
+
+@Data
 public class Order {
+    private Customer to;
+    private Warehouse from;
+    private int distance;
```

Comment:
distance is not really needed, is it?

Reply:
Also, when an order is placed, it is an order for an item (resource in your system), but Order in here is just from to info